src/runtime/virtcontainers/qemu.go

```diff
@@ -667,6 +672,18 @@ func (q *qemu) CreateVM(ctx context.Context, id string, network Network, hypervi
 		qemuConfig.Devices = q.arch.appendPCIeRootPortDevice(qemuConfig.Devices, hypervisorConfig.PCIeRootPort, memSize32bit, memSize64bit)
 	}
 
+	if q.config.Debug && !qemuConfig.Knobs.Daemonize {
```

Comment:
Why check `q.config.Debug`? Don't we always want the qemu error log?

Reply:
> Why check `q.config.Debug`? Don't we always want the qemu error log?

I've assumed this was wanted since it is checked in the existing code for `-D qemu.log`, but I'd be glad to always get QEMU errors if you think it is preferable.

Reply:
I'll remove the check in the next version.